### PR TITLE
UI:Unit tests

### PR DESCRIPTION
**Pull request by deetz99**
update unit tests
cleanup some unused components

### CODE REVIEW

File: web/site/components/Sbc/DatePicker.vue
Comment (deetz99) [inline web/site/components/Sbc/DatePicker.vue:1]: This is not a new component. Im removing the old datepicker components, and renaming the new one from SbcDatePicker2 -> SbcDatepicker

File: web/site/components/Sbc/Datepicker2.vue
Comment (deetz99) [inline web/site/components/Sbc/Datepicker2.vue:1]: deleted because its renamed to SbcDatepicker

File: web/site/components/Sbc/Inputs/DateSelect.vue
Comment (deetz99) [inline web/site/components/Sbc/Inputs/DateSelect.vue:1]: deleted cause im not using 2 components for the datepicker anymore

File: web/site/components/content/ButtonDownloadSpec.vue
Comment (deetz99) [inline web/site/components/content/ButtonDownloadSpec.vue:1]: this is from sbc api gateway

File: web/site/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@nuxtjs/eslint-config-typescript": "^12.1.0",
     "@nuxtjs/eslint-module": "^4.1.0",
     "@pinia-plugin-persistedstate/nuxt": "^1.2.0",
+    "@pinia/testing": "^0.1.3",
```
Comment (deetz99) [inline web/site/package.json:28]: used for mocking stores in component tests

File: web/site/tests/unit/components/ContentBusinessEmail.test.ts
Comment (deetz99) [inline web/site/tests/unit/components/ContentBusinessEmail.test.ts:1]: test the business email nuxt content component renders correctly based on store values

File: web/site/tests/unit/components/ContentDocumentDownload.test.ts
Comment (deetz99) [inline web/site/tests/unit/components/ContentDocumentDownload.test.ts:1]: test the document download nuxt content component renders correctly based on store values
- should only render if theres documents in the annual report object
- clicking the button calls the handleDocumentDownload method
- etc

File: web/site/tests/unit/components/SbcAlert.test.ts
Comment (deetz99) [inline web/site/tests/unit/components/SbcAlert.test.ts:1]: - test only 1 alert is rendered at a time
- test internal server error alert renders even if theres other alert ahead of it
- test it only renders when theres a matching category in the props

File: web/site/tests/unit/components/SbcDatepicker.test.ts
Comment (deetz99) [inline web/site/tests/unit/components/SbcDatepicker.test.ts:1]: testing the new datepicker compoennt

File: web/site/tests/unit/stores/alert.test.ts
Comment (deetz99) [inline web/site/tests/unit/stores/alert.test.ts:1]: test alert store, addAlert method, $reset, hasAlerts computed

File: web/site/tests/unit/stores/tos.test.ts
Comment (deetz99) [inline web/site/tests/unit/stores/tos.test.ts:1]: terms of service store methods, error handling, etc

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-253-nvlvq58j.web.app

**Comment by deetz99** [comment]
dev db needs to be migrated still

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**